distributed/cli/tests/test_dask_scheduler.py

```diff
@@ -97,24 +104,31 @@ def test_dashboard(loop):
 
 def test_dashboard_non_standard_ports(loop):
     pytest.importorskip("bokeh")
-
+    port1 = open_port()
+    port2 = open_port()
     with popen(
-        ["dask-scheduler", "--port", "3448", "--dashboard-address", ":4832"]
+        [
+            "dask-scheduler",
+            "--port",
+            str(port1),
```

Comment:
nit: I find f-strings marginally neater here

```suggestion
            f"--port={port]",
```

Reply:
agreed. I was not entirely sure if our popen would handle this properly but it does 🎉 

I changed most occurences with some regex replacements. might have missed a few but will not go back for this

Reply:
👍